learn/advanced/asynchronous_updates.md

```diff
@@ -1,15 +1,26 @@
 # Asynchronous updates
 
-MeiliSearch is an **asynchronous API**. It means that the API does not behave as you would typically expect when handling the request's responses.
+Index updates are processed **asynchronously**. This means that update requests are not handled as soon as they are received—instead, MeiliSearch places these operations in a queue and processes them in the order they were received.
 
-Some operations are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains the identifier to track the execution of the operation.
+## Which operations are async?
 
-## Async flow
+Every operation that might take a long time to be processed is handled asynchronously.
 
-- When making a write request (_create/update/delete_) against the search engine, it stores the operation received in a queue and returns an `updateId`. With this id, the operation update is trackable.
-- Each update received is treated following the order it has been received.
-- You can get the update status on the [`/updates`](/reference/api/updates.md) route.
-- Processed updates are marked as processed and kept in the operation list (available at `/indexes/:index_uid/updates`). They won't be deleted.
+For example, it is possible updating the `filterableAttributes` index setting will require as much time as generating a new index, so MeiliSearch puts it in the update queue and processes the update request as soon as the instance can allocate the adequate resources.
```

Comment:
```suggestion
For example, updating the `filterableAttributes` index setting will require as much time as re-indexing all your documents from this index. Because of that MeiliSearch adds it in the update queue and processes the update request as soon as the instance can allocate the adequate resources.
```

> and processes the update request as soon as the instance can allocate the adequate resources

I'm not sure this is accurate. Is it not processed once no other updates in the index are being processed? @ManyTheFish

Reply:
```suggestion
For example, updating the `filterableAttributes` index setting will require as much time as re-indexing all the documents in this index. Because of that, MeiliSearch adds your update request to the update queue and processes it as soon as possible.
```